examples/python-cicd-with-cloudbuilder/README.md

```diff
@@ -53,70 +72,84 @@ my_module/__init__.py        1      0   100%
 my_module/my_module.py       4      0   100%
 --------------------------------------------
 TOTAL                        5      0   100%
-
-
 ```
 
-Now that our tests are working locally, we want to configure GCP to run them every time we push new code.
+Now that your tests are working locally, you can configure Cloud Builder to run them every time you push new code.
+
+## 3. Building a Python Cloud Build Container
+To run Python based tests in cloud builder you need a Python [cloud builder](https://cloud.google.com/cloud-build/docs/cloud-builders). 
```

Comment:
"Python-based" is what you want because this is now an adjective but you can just ignore this :).

Reply:
:)